### ISSUE TITLE
[Bug] - Current Copr (Fedora) default configuration broken

### ISSUE THREAD

**Issue by tim77** [amazonlinux/amazon-linux-2023#643]
**Describe the bug**
Current [Copr](https://docs.fedoraproject.org/en-US/infra/sysadmin_guide/copr/) (Fedora) default configuration broken.

**To Reproduce**
1. Try to add Copr repo for Amazon Linux 2023. [Example](https://copr.fedorainfracloud.org/coprs/atim/shadowsocks-rust/) for testing purposes.
2. `sudo dnf copr enable atim/shadowsocks-rust`
3. See error:
```
Error: It wasn't possible to enable this project.
Repository 'epel-2023-x86_64' does not exist in project 'atim/shadowsocks-rust'.
Available repositories: 'fedora-40-x86_64', 'fedora-37-aarch64', 'fedora-38-aarch64', 'epel-7-x86_64', 'fedora-rawhide-x86_64', 'fedora-37-x86_64', 'epel-9-aarch64', 'fedora-40-aarch64', 'amazonlinux-2023-aarch64', 'epel-8-aarch64', 'fedora-39-x86_64', 'fedora-38-x86_64', 'centos-stream-9-aarch64', 'amazonlinux-2023-x86_64', 'fedora-rawhide-aarch64', 'epel-8-x86_64', 'epel-9-x86_64', 'fedora-39-aarch64', 'centos-stream-9-x86_64'
```

**Expected behavior**
Copr repo should successfully added without any additional tweaks.

**Additional context**
Current configuration point to  `epel-2023-x86_64` but the correct one is `amazonlinux-2023-x86_64`.

**Comment by daniejstriata** [comment]
This issue is not with AL2023 itself. The repo you try to add, incorrectly requires EPEL9 as a dependency.

**Comment by tim77** [comment]
The proper fix is the `amazonlinux-release` needs to ship a `/usr/share/dnf/plugins/copr.vendor.conf` file that forces the plugin to select Amazon Linux 2023.

**Comment by daniejstriata** [comment]
@tim77 You are correct. I was able to replicate it with another copr repo. It would appear that the dnf copr plugin is being used without considering that it is not on a Fedora/RHEL host.
